macros

```diff
@@ -77,16 +71,7 @@
 	mandir=%{?buildroot:%{buildroot}}%{_mandir} \\\
 	infodir=%{?buildroot:%{buildroot}}%{_infodir} \\\
   install
-
-%make_install \
-  %{__make} \\\
-	DESTDIR=%{?buildroot:%{buildroot}} \\\
-	INSTALL_ROOT=%{?buildroot:%{buildroot}} \\\
-  install \
-  rm -f %{?buildroot:%{buildroot}}%{_infodir}/dir \
-  find %{?buildroot:%{buildroot}} -regex ".*\\.la$" | xargs rm -f -- \
-  %{!?keepstatic:find %{?buildroot:%{buildroot}} -regex ".*\\.a$" | xargs rm -f --}  
```

Comment:
File installation being such a basic operation on rpm packaging, I could expect some third party software also failing to build if it's just abruptly removed while not getting big wins either on the removal. 

If removed, could first make it warn for a couple of years.

Reply:
Please read the bug report.. The current macro doesn't allow passing of arguments which breaks the packaging or requires workaround.
No package will break from removing the macro except those that incorrectly used this macro
such as those that did not package static libraries but build them anyway or those that did not use the install macro appropriate for qmake as the sdk already suggests.

Any package can be adjusted to use the upstream macro without breaking the build on  older Sailfish OS releases.

Reply:
Ah right Missed that this is overriding rpm's default make_instal. Alright then, makes sense.